cypress/support/commands.ts

```diff
@@ -11,353 +11,277 @@ Cypress.Commands.add("clearIndexedDb", () => {
     });
 });
 
-Cypress.Commands.add("signup", () => {
-    cy.clearCookies();
-    cy.clearLocalStorage();
-    cy.clearIndexedDb();
+Cypress.Commands.add("clearEmails", () => {
+    return cy.request("DELETE", "http://localhost:1080/email/all");
+});
 
-    cy.visit("/");
+Cypress.Commands.add("getCodeFromEmail", (options: any = {}) => {
+    const getCode = () => {
+        return cy.request("http://localhost:1080/email").then((res) => {
+            const latest = res.body.sort((a, b) => (a.time > b.time ? -1 : 1))[0];
+            if (!latest) {
+                return null;
+            }
+            const matchCode = latest.text.match(/(\d{6})/);
+            return matchCode && matchCode[1];
+        });
+    };
+
+    const resolveValue = () => {
+        getCode().then((value) => {
+            // @ts-ignore
```

Comment:
Apparently the `cy.verifyUpcomingAssertions` command is missing from the types, hence the `@ts-ignore`